workspaces/ui/src/services/TestingService.js

```diff
@@ -89,18 +93,30 @@ export async function createExampleTestingService(exampleId = 'todo-report') {
 
 // Might belong in a (View)Model somewhere
 export function queriesFromEvents(events) {
-  const { contexts } = opticEngine.com.useoptic;
-  const { RfcServiceJSFacade } = contexts.rfc;
+  const {contexts} = opticEngine.com.useoptic;
+  const {RfcServiceJSFacade} = contexts.rfc;
   const rfcServiceFacade = RfcServiceJSFacade();
   const eventStore = rfcServiceFacade.makeEventStore();
   const rfcId = 'testRfcId';
 
   // @TODO: figure out if it's wise to stop the parsing of JSON from the response, to prevent
   // parse -> stringify -> parse
-  eventStore.bulkAdd(rfcId, JSON.stringify(events));
+  const stringifiedEvents = JSON.stringify(events);
+  eventStore.bulkAdd(rfcId, stringifiedEvents);
   const rfcService = rfcServiceFacade.makeRfcService(eventStore);
   const queries = Queries(eventStore, rfcService, rfcId);
   const rfcState = rfcService.currentState(rfcId);
 
-  return { queries, rfcState };
+  const SimulatedContext = (props) => (
+    <InitialRfcCommandsStore
+      instance="the one in simulated command context"
+      rfcId={rfcId}
+      initialEventsString={stringifiedEvents}>
+      <ImmutableRfcStore>
+        {props.children}
+      </ImmutableRfcStore>
+    </InitialRfcCommandsStore>
+  );
+
+  return {queries, rfcState, SimulatedContext};
 }
```

Comment:
I think you might have been working of an older branch, because all of this and formatting changes to `TestingService` and `TestingDashboardContext`  re-appeared! Can't think how that would be related, so I'll go ahead and revert those changes.

Reply:
I was pretty confused about why this would happen, but I think the answer is in the git tree.
Here's the commits:

![Screenshot 2020-04-17 at 11 44 39](https://user-images.githubusercontent.com/857549/79561311-6de87280-80a9-11ea-883a-b4a1e5c024db.png)

The orange line is the current branch of this PR, as you can see, the changes made that are missing here never got merged to it. @acunniffe I suspect you might have merged the PR locally, rather than through Github, and never pulled down my commits!

From there, I don't think you went back to `feature/testing-dashboard` and pulled the latest version from Github either, but instead kept on merging those changes back in. 

![Screenshot 2020-04-17 at 12 30 53](https://user-images.githubusercontent.com/857549/79561513-bf90fd00-80a9-11ea-8025-27b11c4e4371.png)

To prevent this in the future, I'd suggest pulling the "main development" branch and branching from that, when starting a new branch. That way, you make sure your new changes are on the latest "stable" development version. Also, merging PRs are best done through the Github interface; it even allows rebasing or squashing as 1 commit, rather than a merge!

Reply:
Fixed with 10b8781193eddcb02cb97794373c94e86f1865b8. @acunniffe be sure to pull these changes this time 😂.

Reply:
Thanks for walking me through exactly all missteps re: gitflow. I'm still getting used to this process and appreciate all the guidance.